docs/snap-actions-playbook.md

```diff
@@ -0,0 +1,73 @@
+# Snap actions playbook
+
+This document describes how to interact with the Tailscale snap itself for performing various tasks.
+
+## How to view the service logs
+
+The logs can be viewed through `snap logs`:
+
+```bash
+sudo snap logs tailscale
+```
+
+For more lines:
+
+```bash
+sudo snap logs -n 100 tailscale
+```
+
+## How to manage the Tailscale service
+
+To stop the Tailscale service:
+
+```console
+$ sudo snap stop tailscale
+Stopped.
+```
+
+While the service is stopped, the `tailscale` client command is expected to fail with the following message:
+
+```console
+$ tailscale status
+failed to connect to local tailscaled; it doesn't appear to be running (sudo systemctl start tailscaled ?)
+```
+
+To start the Tailscale service again:
+
+```console
+$ sudo snap start tailscale
+Started.
+```
+
+The `tailscale` client command should be working again.
+
+The service can also be restarted in a single command:
+
+```console
+$ sudo snap restart tailscale
+Restarted.
+```
+
+Stopping and starting or restarting the Tailscale service should *not* result in needing to re-login.
+
+## How to update Tailscale
+
+To update Tailscale on a machine, refresh through snap:
+
+```console
+$ sudo snap refresh tailscale
+tailscale ... refreshed
```

Comment:
Should we update the output now?

Reply:
Hmm I'm not actually seeing the post-refresh warning :thinking:

Reply:
😕 not sure if the text is cleared by the cli, anyways, resolving it for now as the output is reflecting the true.

Reply:
I assumed earlier it was simply because I couldn't test refreshing with a local snap, but maybe something else is going on. :thinking: Do you have any ideas?